data/test/index.js

```diff
@@ -29,10 +29,6 @@ import {
 	toAsyncIterable,
 } from '../';
 
-jest.mock( '@wordpress/utils', () => ( {
```

Comment:
Would be neat to have something like our ESLint `deprecated` removal rules, except for things like this which aren't necessarily "deprecated", but are related to a deprecation or should otherwise be removed by a certain version. Maybe a comment pattern similar to [ESLint's inline configuration](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments), e.g. `/* REMOVEME: v3.0.0 */`.

Reply:
Yes, it would help to remove it together with the code that was deprecated. Another option would be to include `toHaveWarned` in the code which would fail when deprecation gets removed. Unless I missed something.